C/capnproto/build_tarballs.jl

```diff
@@ -41,7 +39,7 @@ fi
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = expand_cxxstring_abis(supported_platforms())
```

Comment:
This this change?  It doesn't look like this is needed at all
```suggestion
platforms = supported_platforms()
```